redirects.js

```diff
@@ -25,4 +25,46 @@ module.exports = [
     destination: '/terraform/enterprise/sentinel/:path*',
     permanent: false,
   },
+  // Redirects for restructured Terraform Plugin Framework docs for GA release of the Framework
+  // - https://github.com/hashicorp/terraform-plugin-framework/pull/554
+  {
+    source: '/terraform/plugin/framework/schemas',
+    destination: '/terraform/plugin/framework/handling-data/schemas',
+    permanent: false,
```

Comment:
Should the redirects here be permanent, or will the content move back to the old paths at some point?

Reply:
I don't think the content will revert back to the old paths so I've marked all of the redirects as `permanent: true`.